astro/src/content/docs/get-started/start-here/index.mdx

```diff
@@ -0,0 +1,196 @@
+
+Navigate to <Breadcrumb>Applications</Breadcrumb> and then edit the ChangeBank application. Navigate to the <Breadcrumb>Security</Breadcrumb> tab and then the <Breadcrumb>Passwordless login</Breadcrumb> section.
+
+Click the <InlineField>Enable</InlineField> toggle.
+
+![Enable the magic link login.](/img/docs/get-started/start-here/enable-magic-link-login.png)
+
+Click the blue 'save' button to save your configuration change.
+
+![Save the application.](/img/docs/get-started/start-here/save-application.png)
+
+Now, log out of the admin UI.
+
+[Visit your application](http://localhost:8080/) and click the <InlineUIElement>Login</InlineUIElement> button.
+
+Scroll down if needed. You'll see a 'Login With Magic Link' button (TODO might not be that text).
+
+![Login screen with magic link enabled.](/img/docs/get-started/start-here/login-screen-magic-link-enabled.png)
+
+Click on it and enter your user's email address (`richard@example.com` in the screenshot below) and click <InlineUIElement>Send</InlineUIElement>
+
+![After entering users email address.](/img/docs/get-started/start-here/login-screen-magic-link-enter-email.png)
+
+Now you can visit the [email inbox](http://localhost:1080) and you'll see your magic link email. 
+
+![Email inbox with magic link message.](/img/docs/get-started/start-here/email-inbox-magic-link-message.png)
+
+Click on the link, and you'll be logged into your application.
+
+### What Did You Just Do?
+
+You enabled Magic Link login for your application without touching a single line of code.
+
+Congrats!
+
+Let's talk about next steps.
+
+## Quickstarts
+
+Looking for an example in a particular language? Quickstarts walk you through the Launch and Delegate steps, leaving you with a running application in your framework or language of choice, to which you can add functionality.
+
+If you have Docker installed and one of over 20 frameworks and languages, you can run through [our quickstarts](/docs/quickstarts).
+
+Each quickstart includes a complete runnable application and a shell application you can build out by following the step-by-step integration instructions.
+
+## Next Steps
+
+* Learn more about [other identity providers, including SAML, OIDC and Facebook](/docs/lifecycle/authenticate-users/identity-providers/).
+* Learn about [passwordless options including magic links and passkeys](/docs/lifecycle/authenticate-users/passwordless/).
+* Enable [self-service registration](/docs/lifecycle/register-users/basic-registration-forms) to let users sign up for your application.
+* Enable [multi-factor authentication (MFA)](/docs/lifecycle/authenticate-users/multi-factor-authentication). TOTP MFA is free, but other MFA methods require a paid license.
+* Sign up for a [trial FusionAuth Cloud instance](https://account.fusionauth.io/account/free-trial/start) which comes with a free Starter plan to explore [other paid features](/docs/get-started/core-concepts/premium-features).
```

Comment:
We may want to remove this, depending on free trial support (see https://inversoft.slack.com/archives/C051S8N8E/p1735592285725199 )

Reply:
I'll ask Andy about this one